src/adapters/rocket/RETHAdapterV1.sol

```diff
@@ -87,10 +90,21 @@ contract RETHAdapterV1 is ITokenAdapter, MutexLock {
         // Transfer the rETH from the message sender.
         SafeERC20.safeTransferFrom(token, msg.sender, address(this), amount);
 
-        // Burn the rETH to receive ETH.
-        uint256 startingEthBalance = address(this).balance;
-        IRETH(token).burn(amount);
-        uint256 receivedEth = address(this).balance - startingEthBalance;
+        // Set up and execute uniswap exchange
+        SafeERC20.safeApprove(token, uniswapRouterV3, amount);
+        ISwapRouter.ExactInputSingleParams memory params =
+            ISwapRouter.ExactInputSingleParams({
+                tokenIn: token,
+                tokenOut: underlyingToken,
+                fee: 3000,
+                recipient: address(this),
+                deadline: block.timestamp,
+                amountIn: amount,
+                amountOutMinimum: amount * 970 / 1000,
```

Comment:
lets just set this to 0 since we're checking `minimumAmountOut` inside the `Alchemist._unwrap()` function